guides/common/modules/proc_adding-a-vmware-connection-to-server.adoc

```diff
@@ -4,8 +4,9 @@
 Use this procedure to add a VMware vSphere connection in {ProjectServer}'s compute resources.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-adding-vmware-vsphere-connection_{context}[].
 
-Ensure that the host and network-based firewalls are configured to allow {Project} to vCenter communication on TCP port 443.
-Verify that {Project} is able to resolve the host name of vCenter and vCenter is able to resolve {ProjectServer}'s host name.
+.Prerequisites
+* Ensure that the host and network-based firewalls are configured to allow communication from {ProjectServer} to vCenter on TCP port 443.
+* Verify that {ProjectServer} is able to resolve the host name of vCenter and vCenter is able to resolve the host name of {ProjectServer}.
```

Comment:
Ensure that {ProjectServer} and vCenter can resolve each other's host names.

Reply:
Applied!